### ISSUE TITLE
pavucontrol: segfault when exiting with CTRL-Q

### ISSUE THREAD

**Issue by blehel** [void-linux/void-packages#29751]
### System

* xuname: Void 5.11.9_1 x86_64 GenuineIntel notuptodate rrFFF
* package: pavucontrol-4.0_1

### Expected behavior

Pressing CTRL-Q exits cleanly.

### Actual behavior

Closing the window works without problem.
Pressing CTRL-Q to exit triggers a segfault.

Running through gdb:
```
Thread 1 "pavucontrol" received signal SIGSEGV, Segmentation fault.
0x00007ffff7e00a01 in Gtk::Main::quit() () from /usr/lib/libgtkmm-3.0.so.1
>(gdb) bt
#0  0x00007ffff7e00a01 in Gtk::Main::quit() () from /usr/lib/libgtkmm-3.0.so.1
#1  0x000055555559df4a in ?? ()
#2  0x00007ffff7e6a845 in Gtk::Widget_Class::key_press_event_callback(_GtkWidget*, _GdkEventKey*) () from /usr/lib/libgtkmm-3.0.so.1
#3  0x00007ffff747b6b8 in ?? () from /usr/lib/libgtk-3.so.0
#4  0x00007ffff70452ee in ?? () from /usr/lib/libgobject-2.0.so.0
#5  0x00007ffff705d1b9 in g_signal_emit_valist () from /usr/lib/libgobject-2.0.so.0
#6  0x00007ffff705e04f in g_signal_emit () from /usr/lib/libgobject-2.0.so.0
#7  0x00007ffff74259a4 in ?? () from /usr/lib/libgtk-3.so.0
#8  0x00007ffff72d891f in ?? () from /usr/lib/libgtk-3.so.0
#9  0x00007ffff72da463 in gtk_main_do_event () from /usr/lib/libgtk-3.so.0
#10 0x00007ffff69af785 in ?? () from /usr/lib/libgdk-3.so.0
#11 0x00007ffff69e32d2 in ?? () from /usr/lib/libgdk-3.so.0
#12 0x00007ffff6f54cfb in g_main_context_dispatch () from /usr/lib/libglib-2.0.so.0
#13 0x00007ffff6f54fa8 in ?? () from /usr/lib/libglib-2.0.so.0
#14 0x00007ffff6f5505f in g_main_context_iteration () from /usr/lib/libglib-2.0.so.0
#15 0x00007ffff6684785 in g_application_run () from /usr/lib/libgio-2.0.so.0
#16 0x000055555558bf69 in main ()
```

**Comment by ericonr** [comment]
Problem is known, iirc the patch has to be rebased, and upstream isn't making a release yet.

It happens because they are using some deprecated stuff in a bad manner, I think.

**Comment by ericonr** [comment]
Fixed in faf771ea10da472e43ae2dc836b7feec59f49896

Just needed some love for the patch to apply.
